distributed/core.py

```diff
@@ -89,13 +95,14 @@ class Server(object):
     default_ip = ''
     default_port = 0
 
-    def __init__(self, handlers, stream_handlers=None, connection_limit=512,
+    def __init__(self, handlers, blocked_handlers=None, stream_handlers=None, connection_limit=512,
                  deserialize=True, io_loop=None):
         self.handlers = {
             'identity': self.identity,
             'connection_stream': self.handle_stream,
         }
         self.handlers.update(handlers)
+        self.blocked_handlers = blocked_handlers or []
```

Comment:
```suggestion
        if blocked_handlers is None:
            blocked_handlers = dask.config.get('distributed.%s.blocked-handlers' % type(self).lower(), [])
        self.blocked_handlers = blocked_handlers
```

And then can you add `blocked-handlers` entries to `distributed/distributed.yaml` under the `distributed.worker` and `distributed.scheduler` sections?  (My hope is that editing this file will be self explanatory if you haven't done it before)

Reply:
Nice, I like that - allows for changing this behavior entirely from config, which was an initial desire.  Should I add a test for this init behavior (pulling from config)?

Reply:
Ping on the following:

> And then can you add blocked-handlers entries to distributed/distributed.yaml under the distributed.worker and distributed.scheduler sections? (My hope is that editing this file will be self explanatory if you haven't done it before)

Relevant file is here: https://github.com/dask/distributed/blob/master/distributed/distributed.yaml

A test would be welcome, but is not obligatory

Reply:
Yup sorry for the delay; updated config + a test for the scheduler.